### DISCUSSION TITLE
Add a "Reconciled" status for transactions

### DISCUSSION THREAD

**Discussion by Deguello** [actualbudget/releases#71] [Feature ideas]
It would be nice to have a Reconciled status for transactions, in addition to Cleared and Uncleared.

Further, it would be nice to have the ability (optionally) to hide Reconciled transactions in the register. Generally speaking, once a transaction is reconciled, I'm done with it and never need to see it again in the context of the register.

**Comment by bpaulien** [comment]
I would love to have a "reconciled" status on a transaction.  And once reconciled, there should be logic that at least pops up a warning if an amount is inadvertently changed on a reconciled transaction.  Because that would mess up the reconciliation for the account, and who knows how far back you'd have to go to find the discrepancy.

**Comment by purpledogeatsgreenpoop** [comment]
Just trying to bounce this one up to the top.  I miss the ability to hide all reconciled entries - just made it easier to focus on what needs to be done and move one.

**Comment by Pikafu** [comment]
Agreed with all of you. As a workaround, I have been creating a $0 transaction with a note "RECONCILED" to indicate when was the last point in time I reconciled.

**Reply by Silvenga** [reply]
That's not a bad idea actually. I'm going to start doing that. 😹

**Reply by MiriamNZ** [reply]
Add the account balance to the notes. If the running balance is different, you know you will have to go further back.

**Comment by Kidglove57** [comment]
Personally I am totally ambivalent to the idea of hiding transactions - I did not really like that in the old YNAB - and of course we want Actual to be special in its own right and not simply a YnAB clone.

However, I am all for being able to lock a transaction once reconciled. At the moment I clear transactions and then simply compare the cleared balance (click to the right of balance at top of screen to reveal this) to the cleared balance in my banking app. I tend to do this daily so I always feel I am up to date. In effect, I am using the cleared balance to reconcile daily.

**Comment by Rick-Thomas** [comment]
Let me share my workflow for this...

I use all three concepts...   1) Not cleared  2) Cleared  3) Reconciled...    

1. The "**_not cleared_**" is obvious, the Institution (Bank or CC or whatever) hasn't received this transaction...   
2. The "**_cleared_**" is a mark that the institution has seen this transaction...   
3. The "**_Reconciled_**" is when, at the end of the month, I have received a formal document from the institution (eg. bank statement), with all movements for that month...   In this final instance of "**_Reconciled_**" I do not want any changes to the transaction because it is locked with the bank...   At this point I do not requite these transactions to be displayed every time I go to the account...   I like the YNAB concept (didin't Quicken have it also???) of Hide/Unhide Reconciled transactions...   It cleans the "page", and it has permitted me to suddenly detect a transaction which is some months old and is still sitting there not reconciled!!!

**Reply by Deguello** [reply]
I did it like that for ages. At this point, for me, with a reconciled status available, a cleared status exists only for as long as it takes me to make my cleared balance match the bank's. Then, ideally, an app lets me reconcile an amount (the currently cleared balance, by default) for a date (today's date, by default) and it marks those transactions cleared and hides them.

I haven't looked at an actual statement to reconcile it in years now.

**Reply by Kidglove57** [reply]
Yes. I don't reconcile to statements either. Simply because I always have the latest balance  on my phone and from there I can see the transactions that have cleared my bank and those that are marked as pending. In old days with YNAB I did rather enjoy using this to reconcile and lock transactions almost daily.

**Comment by mkemen** [comment]
Add my vote to have the "Cleared" and the "Reconciled" as two statuses.   The ability to hide reconciled transactions also makes my life a bit easier.

**Comment by MiriamNZ** [comment]
I created a payee called 'reconciled' for the dummy transaction. I also put the account total in the notes. (In case of changes south of the reconciled line. )
